src/translator/definitions.h

```diff
@@ -28,4 +28,21 @@ typedef AlignedVector<char> AlignedMemory;
 } // namespace bergamot
 } // namespace marian
 
+// @TODO at the moment the usage of string_view in this repository is a hot mess and a disaster waiting to happen.
+// ssplit uses std::string_view if the compiler supports c++17, else falls back to c++11 and absl::string_view
+// bergamot-translator uses, depending on the source file std::string_view (which will break if ssplit-cpp uses
+// absl::string_view) and marian::string_view which is an export of (confusingly) the sentencepiece module that
+// marian has. marian::string_view is our addition to the marian fork, which will make merging even nicer. Not.
+// This is just an ugly patchwork that allos gcc5, our lowest targetted gcc to run. We don't actually try to run
+// on older compilers.
```

Comment:
So, I was convinced what Nick is doing here is the minimum invasive edit, restricting to what this PR describes it is doing. We can delete the files in another one.

ssplit-cpp I think still maintains an `std::string_view` dependency, far as I understand, although it can change.